lxd/db/images.go

```diff
@@ -1060,6 +1060,27 @@ func (c *Cluster) GetPoolsWithImage(imageFingerprint string) ([]int64, error) {
 	return poolIDs, nil
 }
 
+// GetPoolsWithRemoteImage gets the IDs of all storage pools on which a given image exists.
```

Comment:
Probably worth expending a bit on this comment.

Looks like you're getting a list of pool IDs for pools backed by remote storage which hold a particular image.